File: Chapter05/code/Exercise05_18.scala
:paste
gpa
  .cube($"school", $"class")
  .count()
  .sort(asc("school"), asc("class"))
  .show()

:paste
gpa
  .rollup($"school", $"class")
  .count()
  .sort(asc("school"), asc("class"))
  .show()

File: Chapter05/code/Exercise05_18.py
gpa \
  .cube(col("school"), col("class")) \
  .count() \
  .sort(asc("school"), asc("class")) \
  .show() 

gpa \
  .rollup(col("school"), col("class")) \
  .count() \
  .sort(asc("school"), asc("class")) \
  .show()
